clipboard/ios/Plugin/ClipboardPlugin.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import Capacitor
+
+@objc(ClipboardPlugin)
+public class ClipboardPlugin: CAPPlugin {
+    private let implementation = Clipboard()
+
+    @objc func read(_ call: CAPPluginCall) {
+        let result = implementation.read()
+        
+        if !result.isEmpty {
+            call.resolve(result)
+        }
+    }
+    
+    @objc func write(_ call: CAPPluginCall) {
+        var success : Bool = false
+        
+        if let string = call.options["string"] as? String {
+            implementation.write(content: string, ofType: Clipboard.ContentType.string)
+            success = true
+        } else if let urlString = call.options["url"] as? String {
+            implementation.write(content: urlString, ofType: Clipboard.ContentType.url)
+            success = true
+        } else if let imageBase64 = call.options["image"] as? String {
+            implementation.write(content: imageBase64, ofType: Clipboard.ContentType.image)
+            success = true
+        }
+        
+        if success {
+            call.resolve()
+        } else {
+            call.error("No data provided")
```

Comment:
Shouldn't this be `reject`? My understanding is that we were going to prefer `resolve` & `reject` over `success` & `error`.

Reply:
Will change it now. Do we want to add the reject to the checklist also @dwieeb 
` I have switched all deprecated .success() calls to use .resolve().`